www/front_src/src/components/header/getResourcesUrl.ts

```diff
@@ -0,0 +1,65 @@
+const hostCriterias = { resourceTypes: [{ id: 'host', name: 'Host' }] };
+const serviceCriteria = { resourceTypes: [{ id: 'service', name: 'Host' }] };
+
+interface StatusCriterias {
+  statuses: Array<{ id: string; name: string }>;
+}
+
+const getStatusCriterias = (status): StatusCriterias => {
+  return { statuses: [status] };
+};
+
+const downCriterias = getStatusCriterias({ id: 'DOWN', name: 'Down' });
+const unreachableCriterias = getStatusCriterias({
+  id: 'UNREACHABLE',
+  name: 'Unreachable',
+});
+const upCriterias = getStatusCriterias({ id: 'UP', name: 'Up' });
+const pendingCriterias = getStatusCriterias({ id: 'PENDING' });
+const criticalCriterias = getStatusCriterias({
+  id: 'CRITICAL',
+  name: 'Critical',
+});
+const warningCriterias = getStatusCriterias({ id: 'WARNING', name: 'Warning' });
+const unknownCriterias = getStatusCriterias({ id: 'UNKNOWN', name: 'Unknown' });
+const okCriterias = getStatusCriterias({ id: 'OK', name: 'Ok' });
```

Comment:
All this status information is common in Centreon. Shouldn't we put them somewhere more general so as not to define them several times?

Reply:
I tend to prefer to wait for another use case to generalize any implementation